Sources/UInt128.swift

```diff
@@ -399,7 +399,7 @@ extension UInt128 : FixedWidthInteger {
         
         for numeratorShiftWidth in (0...numeratorBitsToWalk).reversed() {
             remainder <<= 1
-            remainder |= UInt128._bitFromDoubleWidth(at: numeratorShiftWidth, for: dividend)
+            remainder |= ._bitFromDoubleWidth(at: numeratorShiftWidth, for: dividend)
```

Comment:
Should we change this helper function name like you did with the string ones to make it more swiftish?